apps/wallet/src/ui/app/shared/faucet/request-button/index.tsx

```diff
@@ -16,40 +18,90 @@ import type { ButtonProps } from '_app/shared/button';
 
 import st from './RequestButton.module.scss';
 
+//TODO:  Remove post testnet
+const TESTNET_DISCORD_LINK =
+    'https://discord.com/channels/916379725201563759/1037811694564560966';
+
 type FaucetRequestButtonProps = {
     mode?: ButtonProps['mode'];
     trackEventSource: 'home' | 'settings';
 };
 
+function FaucetDiscordLink({
+    mode = 'primary',
+    trackEventSource,
+}: {
+    mode?: ButtonProps['mode'];
+    trackEventSource: 'home' | 'settings';
+}) {
+    return (
+        <ExternalLink
+            className={cl(
+                'flex flex-nowrap max-w-full items-center justify-center rounded-xl px-5 py-3 gap-2 no-underline font-medium text-body',
+                mode === 'primary'
+                    ? 'bg-hero-dark text-white hover:bg-hero'
+                    : 'bg-white text-steel border-solid border-color-steel border hover:border-color-steel-dark hover:text-steel-dark'
+            )}
+            href={TESTNET_DISCORD_LINK}
+            showIcon={false}
+            onClick={() =>
+                trackEvent('DiscordRequestSUIToken', {
+                    props: { source: trackEventSource },
+                })
+            }
+        >
+            Request Testnet SUI on Discord
+            <ArrowUpRight16
+                className={cl(mode === 'primary' ? 'text-white' : 'text-steel')}
+            />
+        </ExternalLink>
+    );
+}
+
 function FaucetRequestButton({
     mode = 'primary',
     trackEventSource,
 }: FaucetRequestButtonProps) {
     const network = useAppSelector(({ app }) => app.apiEnv);
     const networkName = API_ENV_TO_INFO[network].name.replace(/sui\s*/gi, '');
     const mutation = useFaucetMutation();
-    return mutation.enabled ? (
-        <Button
-            mode={mode}
-            onClick={() => {
-                toast.promise(mutation.mutateAsync(), {
-                    loading: <FaucetMessageInfo loading />,
-                    success: (totalReceived) => (
-                        <FaucetMessageInfo totalReceived={totalReceived} />
-                    ),
-                    error: (error) => (
-                        <FaucetMessageInfo error={error.message} />
-                    ),
-                });
-                trackEvent('RequestGas', {
-                    props: { source: trackEventSource, networkName },
-                });
-            }}
-            disabled={mutation.isMutating}
-        >
-            <Icon icon={SuiIcons.Download} className={cl(st.icon, st[mode])} />
-            Request {networkName} SUI Tokens
-        </Button>
+
+    //TODO: remove this TestNet check after testnet
+    return mutation.enabled || network === 'testNet' ? (
```

Comment:
I would just add an if before to check for testnet something like
```
if (is testner) {
    return the external component
}
and the rest as it was
```

Reply:
done